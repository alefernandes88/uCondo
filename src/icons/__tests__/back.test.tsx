import React from 'react'
import { render } from '@testing-library/react-native'
import BackIcon from '../back'

describe('Icons -> Back', () => {
  it('should render with a normal style', () => {
    const { toJSON } = render((
      <BackIcon />
    ))

    expect(toJSON()).toMatchSnapshot()
  })

  it('should render with a custom style', () => {
    const { toJSON } = render((
      <BackIcon
        color="red"
        height={30}
        width={30}
        strokeSize={1.5}
      />
    ))

    expect(toJSON()).toMatchSnapshot()
  })
})
