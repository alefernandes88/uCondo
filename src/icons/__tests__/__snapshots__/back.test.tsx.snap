// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Icons -> Back should render with a custom style 1`] = `
<RNSVGSvgView
  align="xMidYMid"
  bbHeight="30"
  bbWidth="30"
  fill="none"
  focusable={false}
  height={30}
  meetOrSlice={0}
  minX={0}
  minY={0}
  style={
    [
      {
        "backgroundColor": "transparent",
        "borderWidth": 0,
      },
      {
        "flex": 0,
        "height": 30,
        "width": 30,
      },
    ]
  }
  vbHeight={267}
  vbWidth={375}
  width={30}
>
  <RNSVGGroup
    fill={null}
    propList={
      [
        "fill",
      ]
    }
  >
    <RNSVGPath
      d="M0 0H375V267H0V0Z"
      fill={
        {
          "payload": 4294901760,
          "type": 0,
        }
      }
      propList={
        [
          "fill",
        ]
      }
    />
  </RNSVGGroup>
</RNSVGSvgView>
`;

exports[`Icons -> Back should render with a normal style 1`] = `
<RNSVGSvgView
  align="xMidYMid"
  bbHeight="267"
  bbWidth="375"
  fill="none"
  focusable={false}
  height={267}
  meetOrSlice={0}
  minX={0}
  minY={0}
  style={
    [
      {
        "backgroundColor": "transparent",
        "borderWidth": 0,
      },
      {
        "flex": 0,
        "height": 267,
        "width": 375,
      },
    ]
  }
  vbHeight={267}
  vbWidth={375}
  width={375}
>
  <RNSVGGroup
    fill={null}
    propList={
      [
        "fill",
      ]
    }
  >
    <RNSVGPath
      d="M0 0H375V267H0V0Z"
      fill={
        {
          "payload": 4284621968,
          "type": 0,
        }
      }
      propList={
        [
          "fill",
        ]
      }
    />
  </RNSVGGroup>
</RNSVGSvgView>
`;
